gen/src/generate/ucd/normal.rs

```diff
@@ -147,10 +191,11 @@ impl CanonicalCompositionData {
             }),
         )
     }
-}
 
-impl<'a, 'b> From<&'a CanonicalDecompositionData<'b>> for CanonicalCompositionData {
```

Comment:
What happened is I needed to take two things, the decomposition and the exclusion.

I suppose we could have a `impl From<(Decomposition, Exclusions)>` (or just call it `new`).